sympy/integrals/manualintegrate.py

```diff
@@ -1553,9 +1588,32 @@ def eval_reciprocal_sqrt_quadratic(a, b, c, integrand, x):
     return log(2*sqrt(c)*sqrt(a+b*x+c*x**2)+b+2*c*x)/sqrt(c)
 
 
+@evaluates(SqrtQuadraticDenomRule)
+def eval_sqrt_quadratic_denom(a, b, c, coeffs: list[Expr], integrand, x):
+    R"""
+    Integrate poly/sqrt(a+b*x+c*x**2) using the recursion formula below. coeffs are coefficients of the polynomial.
+    Let $I_n=\int\frac{x^n}{\sqrt{a+bx+cx^2}}\mathrm{d}x$,
+    then $I_n=\frac{1}{cn}x^{n-1}\sqrt{a+bx+cx^2}-\frac{2n-1}{2n}\frac{b}{c}I_{n-1}-\frac{n-1}{n}\frac{a}{c}I_{n-2}$
```

Comment:
There are a lot of long lines of code in this PR that should be reformatted in some way.

Reply:
Is there a unified standard in SymPy? I think a tool to handle this is ideal.
Moreover, I don’t think latex here should be split into multiple lines as that makes copy-pasting harder.

Reply:
> Is there a unified standard in SymPy?

I don't think that there is a unified standard but generally PEP 8 is a good guide for style.

> I think a tool to handle this is ideal.

Automated tools like Black do not write readable code. They just enforce arbitrary rules like line lengths. It is still for the programmer to take time to think about how to write something that is reasonable.

This particular docstring would most likely fail if included in the html docs because of the unescaped expression in the first line. If this docstring isn't going to be included in the html docs then it's best not to use LaTeX at all because something like this is more readable if written more or less in plain text (e.g. `1/n` rather than `\frac{1}{n}`)

> I don’t think latex here should be split into multiple lines as that makes copy-pasting harder.

Even if I was writing this in LaTeX I would definitely use some spaces somewhere. Ideally what we write in a docstring should be something that is readable by a human rather than something that should be copy-pasted anywhere.

Reply:
This shouldn't be in html docs as it's only for internal usage. Maybe I should just use a block of comment instead of docstring to not import other complexity than what's aimed for in this PR.

Reply:
The same points apply to comments as well as docstrings.

By the way there is actually a style guide for docstrings (which I didn't mention here because this isn't really a proper docstring):
https://docs.sympy.org/dev/guides/contributing/docstring.html